routes/web.php

```diff
@@ -20,6 +20,10 @@
 $router->get('campaigns', 'Legacy\Web\CampaignsController@index');
 $router->get('campaigns/{id}/inbox', 'Legacy\Web\CampaignsController@showInbox');
 $router->get('campaigns/{id}', 'Legacy\Web\CampaignsController@showCampaign')->name('campaigns.show');
+// Create, update, delete campaigns via Rogue.
+$router->post('campaigns', 'CampaignsController@store');
+$router->patch('campaigns/{campaign}', 'CampaignsController@update');
+$router->delete('campaigns/{campaign}', 'CampaignsController@destroy');
```

Comment:
@chloealee will we want `get` here?

Reply:
@katiecrane good q! The `GET /api/v3/campaigns` and `GET /api/v3/campaigns/:campaign_id` routes are all in `routes/api.php`!

Reply:
Can we put these in a separate controller inside [`app/Http/Controllers/Legacy/Web`](https://github.com/DoSomething/rogue/tree/master/app/Http/Controllers/Legacy/Web)? I don't think it will actually cause trouble, since the guard is applied by the middleware group here, but it's a little disorienting that some of the methods in the controller use OAuth and some use HTTP sessions.

Reply:
@DFurnes ah ok! Since there already is a `CampaignsController` in `app/Http/Controllers/Legacy/Web` with an `index` and `showCampaign` method, should I create a new one and call it `CampaignsController` and maybe rename the existing `CampaignsController` to `AshesCampaignsController`?

Reply:
@chloealee I think that's a good plan!